docs/binary_input.md

```diff
@@ -19,11 +19,15 @@ is out of range it will be adjusted to the nearer border. For example, when mode
 - if input shape is [1,90,200,3] it will be resized into [1,100,200,3]
 - if input shape is [1,220,200,3] it will be resized into [1,200,200,3]
 
-Processing the binary image requests requires the model or the custom nodes to accept NHWC layout in BGR color 
+When the endpoint (model or pipeline) shape supports any dimension value (`-1`), the image is not resized and it is enforced to have resolution aligned for entire batch of images.
+
+Resize functionality is also disabled by default for endpoints with layouts different than `...` and `NHWC` (or `N?HWC`, when modified by demultiplexer).
+
+Processing the binary image requests requires the model or the custom nodes have layout compatible with `N...HWC` and BGR color 
```

Comment:
```suggestion
Requests with binary images can be processed for models or the custom nodes with the input layout `NHWC` or `N...HWC`. The models' inputs should use BGR color format.
```

Reply:
This is not true, requests with binary images can be processed for any layout that is compatible with `N...HWC`, as stated in my change.
This means, binary images are supported for any endpoint with layout that can be intersected with `N...HWC`, for example:
- ... (custom node)
- N... (default model layout)
- N?... (default model layout + demultiplexer)
- NH... (--layout NH...)
- NHW... (--layout NHW...)
- NHWC  (--layout NHWC)
- N?HWC (--layout NHWC + demultiplexer)

Reply:
Improved the doc